python/HISTORY.rst

```diff
@@ -2,6 +2,13 @@
 History
 =======
 
+0.19.0 (2022-02-01)
+-------------------
+
+* Reads using "detached" commit-ids w/o specifying a branch or tag name
+* Support for Iceberg views (experimental)
+* Nessie ref-log
```

Comment:
Did we add anything for reflog in python this time?

Reply:
Someone did, no? ;)

Reply:
Oh, right, that was 0.18